tools/tools.go

```diff
@@ -10,6 +10,7 @@ import (
 	_ "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway"
 	_ "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger"
 	_ "github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2"
+	_ "golang.org/x/tools/cmd/goimports"
```

Comment:
Just out of curiosity, do we know why this was added?

Reply:
@bryanchriswhite `ignite` (the latest version) prompted to add this. I realized this wouldn't hurt, so I agreed. :) I assume `ignite` uses `goimports` internally.